e2e/atlas/ldap_test.go

```diff
@@ -180,3 +216,36 @@ func TestLDAP(t *testing.T) {
 		assert.Contains(t, string(resp), "LDAP configuration userToDNMapping deleted")
 	})
 }
+
+func testLDAPVerifyCmd(t *testing.T, cmd *exec.Cmd) string {
+	t.Helper()
+
+	cmd.Env = os.Environ()
+	resp, err := cmd.CombinedOutput()
+	require.NoError(t, err, string(resp))
+
+	a := assert.New(t)
+	var configuration mongodbatlas.LDAPConfiguration
+	if err := json.Unmarshal(resp, &configuration); a.NoError(err) {
+		a.Equal(pending, configuration.Status)
+		a.Equal(ldapBindPassword, configuration.LDAP.BindPassword)
```

Comment:
this is producing a segmentation fault on our tests
```
 [2021/11/08 23:15:08.699] FAILURE: Failed ()
 [2021/11/08 23:15:08.699] === RUN   TestLDAP/Verify
 [2021/11/08 23:15:08.699]     --- FAIL: TestLDAP/Verify (0.12s)
 [2021/11/08 23:15:08.699] panic: runtime error: invalid memory address or nil pointer dereference [recovered]
 [2021/11/08 23:15:08.699] 	panic: runtime error: invalid memory address or nil pointer dereference
 [2021/11/08 23:15:08.699] [signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x59fc0e]
 [2021/11/08 23:15:08.699] goroutine 11 [running]:
 [2021/11/08 23:15:08.699] testing.tRunner.func1.2({0x5c0e80, 0x76a460})
 [2021/11/08 23:15:08.699] 	/opt/golang/go1.17/src/testing/testing.go:1209 +0x24e
 [2021/11/08 23:15:08.699] testing.tRunner.func1()
 [2021/11/08 23:15:08.699] 	/opt/golang/go1.17/src/testing/testing.go:1212 +0x218
 [2021/11/08 23:15:08.699] panic({0x5c0e80, 0x76a460})
 [2021/11/08 23:15:08.699] 	/opt/golang/go1.17/src/runtime/panic.go:1038 +0x215
 [2021/11/08 23:15:08.699] github.com/mongodb/mongocli/e2e/atlas_test.testLDAPVerifyCmd(0xc00017a340, 0xc000136c60)
 [2021/11/08 23:15:08.699] 	/data/mci/dc27fd5560d63c8af9dc6b6760559cb7/src/github.com/mongodb/mongocli/e2e/atlas/ldap_test.go:231 +0x1ae
 [2021/11/08 23:15:08.699] github.com/mongodb/mongocli/e2e/atlas_test.TestLDAP.func1(0x1)
 [2021/11/08 23:15:08.699] 	/data/mci/dc27fd5560d63c8af9dc6b6760559cb7/src/github.com/mongodb/mongocli/e2e/atlas/ldap_test.go:64 +0x212
 [2021/11/08 23:15:08.699] testing.tRunner(0xc00017a340, 0xc000079d70)
 [2021/11/08 23:15:08.699] 	/opt/golang/go1.17/src/testing/testing.go:1259 +0x102
 [2021/11/08 23:15:08.699] created by testing.(*T).Run
 [2021/11/08 23:15:08.699] 	/opt/golang/go1.17/src/testing/testing.go:1306 +0x35a
```

Reply:
The `LDAP` property isn't populated in the `PENDING` repsonse, which is why it's segfaulted. I've removed the assertion against `.LDAP.BindPassword`